### DISCUSSION TITLE
OcR好像正常了

### DISCUSSION THREAD

**Discussion by kevzcr** [shuax/MouseInc.Settings#335] [General]
早上，打开电脑先试了试OCR，发现已经可以使用了，第一时间反馈一下，感谢！

**Comment by yangnis** [comment]
请问你那个设置界面进入正常吗？

**Reply by kevzcr** [reply]
设置界面进不去了，以前还没碰到过这个问题

**Reply by yangnis** [reply]
一直转圈对吧，我看作者好像是刚改了什么东西

**Reply by kevzcr** [reply]
去更新一下就好了  https://dl.shuax.com/MouseInc2.13.0.7z

**Reply by trhy631** [reply]
更新了ocr功能还是不行

**Reply by sunniestboy** [reply]
我更新了2.13.0，ocr功能也还是不行，提示 发生未知错误。

**Reply by trhy631** [reply]
> 去更新一下就好了 https://dl.shuax.com/MouseInc2.13.0.7z

你这个连接是在哪里看到的，为啥官网上没看见这个最新的版本

**Comment by yangnis** [comment]
请问楼主为啥更新检查就提示错误，ocr也不行，我刚刚也手动安装了新的

**Reply by EK-night** [reply]
我刚刚新下载了最新版本的MouseInc2.13.0，OCR还是用不了提示发生未知错误，但是界面进得去了不会一直在加载转圈。

**Reply by kevzcr** [reply]
首先关于ocr, 今早我8点多打开电脑，就已经正常能使用了，我没有进行任何操作，是软件方做出了修改，到现在，一切正常。
第二个关于设置界面进不去，我按照软件作者提供的更新文件，下载替换了我电脑中的文件后，也一切都正常了。
昨天为了Ocr功能，我把系统都重新做了，就是为了排除是不是我个人设置出了什么问题，证明和个人无关，是软件方的问题。

**Reply by EK-night** [reply]
> 首先关于ocr, 今早我8点多打开电脑，就已经正常能使用了，我没有进行任何操作，是软件方做出了修改，到现在，一切正常。 第二个关于设置界面进不去，我按照软件作者提供的更新文件，下载替换了我电脑中的文件后，也一切都正常了。 昨天为了Ocr功能，我把系统都重新做了，就是为了排除是不是我个人设置出了什么问题，证明和个人无关，是软件方的问题。

我分流把ocr的接口走代理了才可以正常用，不走代理用不了

**Reply by uiop321** [reply]
> > 首先关于ocr, 今早我8点多打开电脑，就已经正常能使用了，我没有进行任何操作，是软件方做出了修改，到现在，一切正常。第二个关于设置界面进不去，我按照软件作者提供的更新文件，下载替换了我电脑中的文件后，也一切都正常了。昨天为了Ocr功能，我把系统都重新做了，就是为了排除是不是我个人设置出了什么问题，证明和个人无关，是软件方的问题。
> 
> 我分流把ocr的接口走代理了才可以正常用，不走代理用不了

更新了2.13.0的版本后一切正常了。可以ocr了 。ocr 的接口走代理请问，讲一下步骤。以防以后挂了。

**Reply by EK-night** [reply]
就用代理工具（v2.clash）这类的让接口走代理呀。奇怪为什么我就不可以直接用呢

**Reply by uiop321** [reply]
> 就用代理工具（v2.clash）这类的让接口走代理呀。奇怪为什么我就不可以直接用呢

你看看重启路由器呢，还有就是360修补一下网络连接那个。感觉还是有意想不到的用处。总体感觉和网络是有一定的原有的。

**Reply by EK-night** [reply]
> 你看看重启路由器呢，还有就是360修补一下网络连接那个。感觉还是有意想不到的用处。总体感觉和网络是有一定的原有的。

好的谢谢，我有空再去看下

**Reply by Robertxjf** [reply]
> > > 首先关于ocr, 今早我8点多打开电脑，就已经正常能使用了，我没有进行任何操作，是软件方做出了修改，到现在，一切正常。第二个关于设置界面进不去，我按照软件作者提供的更新文件，下载替换了我电脑中的文件后，也一切都正常了。昨天为了Ocr功能，我把系统都重新做了，就是为了排除是不是我个人设置出了什么问题，证明和个人无关，是软件方的问题。
> > 
> > 
> > 我分流把ocr的接口走代理了才可以正常用，不走代理用不了
> 
> 更新了2.13.0的版本后一切正常了。可以ocr了 。ocr 的接口走代理请问，讲一下步骤。以防以后挂了。

没错，实测不用更新，但是ocr功能API需要走代理了

**Reply by a137933** [reply]
> > > > 首先关于ocr, 今早我8点多打开电脑，就已经正常能使用了，我没有进行任何操作，是软件方做出了修改，到现在，一切正常。第二个关于设置界面进不去，我按照软件作者提供的更新文件，下载替换了我电脑中的文件后，也一切都正常了。昨天为了Ocr功能，我把系统都重新做了，就是为了排除是不是我个人设置出了什么问题，证明和个人无关，是软件方的问题。
> > > 
> > > 
> > > 我分流把ocr的接口走代理了才可以正常用，不走代理用不了
> > 
> > 
> > 更新了2.13.0的版本后一切正常了。可以ocr了 。ocr 的接口走代理请问，讲一下步骤。以防以后挂了。
> 
> 没错，实测不用更新，但是ocr功能API需要走代理了

不更新要这么搞代理？哪里设置

**Reply by ShuaiDaiGuo** [reply]
> > > > > 首先关于ocr, 今早我8点多打开电脑，就已经正常能使用了，我没有进行任何操作，是软件方做出了修改，到现在，一切正常。第二个关于设置界面进不去，我按照软件作者提供的更新文件，下载替换了我电脑中的文件后，也一切都正常了。昨天为了Ocr功能，我把系统都重新做了，就是为了排除是不是我个人设置出了什么问题，证明和个人无关，是软件方的问题。
> > > > 
> > > > 
> > > > 我分流把ocr的接口走代理了才可以正常用，不走代理用不了
> > > 
> > > 
> > > 更新了2.13.0的版本后一切正常了。可以ocr了 。ocr 的接口走代理请问，讲一下步骤。以防以后挂了。
> > 
> > 
> > 没错，实测不用更新，但是ocr功能API需要走代理了
> 
> 不更新要这么搞代理？哪里设置

Clash Windows，主页面，Profiles，右键你自己的Profile，rules，add，Content填 MouseInc.exe，Type选PROCESS-NAME，Proxy or Policy 选你比较稳定的代理服务器，add，save。

**Comment by a137933** [comment]
今天又不行了

**Comment by Eunknight** [comment]
作者ocr用了有道的demo免费接口，疑似有道做了每日IP限制。所以有些人可以，有些人不可以，有些行了又不行，有些不行了又行了。我的公司电脑不可以，家里的可以。
解决这个问题，需要作者开发自定义API接口配置。

**Reply by trhy631** [reply]
你有别的类似的ocr识别软件吗

**Reply by Robertxjf** [reply]
或者作者需要在配置页面加入 **代理设置的选项**，让API走代理端口访问

**Reply by Eunknight** [reply]
> 你有别的类似的ocr识别软件吗

@trhy631 
**utools** 的 **讯飞ocr** 插件。
需要配置自定义个人配置（每日500次调用，有说明），设置快捷键。
体验不如mouseInc的有道翻译接口，但是现在mouseInc有道接口不能用，更新以后其他接口表现太差。
